### PR TITLE
fix names of programming languages

### PR DESCRIPTION
**Pull request by sqs**
- TypeScript and JavaScript (note capital 'S')
- Go is the language name, not Golang

### CODE REVIEW

File: web/src/repogroups/HomepageConfig.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const homepageLanguageList: { name: string; filterName: string }[] = [
     { name: 'Haskell', filterName: 'haskell' },
     { name: 'Html', filterName: 'html' },
     { name: 'Java', filterName: 'java' },
-    { name: 'Javascript', filterName: 'javascript' },
```
Comment (sqs) [inline web/src/repogroups/HomepageConfig.ts:28]: FYI @attfarhan the S should be capitalized in JavaScript and TypeScript

File: cmd/frontend/graphqlbackend/search_suggestions_test.go [hunk 2]
```diff
@@ -301,7 +301,7 @@ func TestSearchSuggestions(t *testing.T) {
 			return &inventory.Inventory{
 				Languages: []inventory.Lang{
 					{Name: "Go"},
-					{Name: "Typescript"},
+					{Name: "TypeScript"},
```
Comment (sqs) [inline cmd/frontend/graphqlbackend/search_suggestions_test.go:304]: FYI @unknwon the S should be capitalized in TypeScript

File: web/src/repogroups/HomepageConfig.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ export const homepageLanguageList: { name: string; filterName: string }[] = [
     { name: 'Haskell', filterName: 'haskell' },
     { name: 'Html', filterName: 'html' },
```
Comment (felixfbecker) [inline web/src/repogroups/HomepageConfig.ts:26]: Shouldn't this be all-caps?
```suggestion
    { name: 'HTML', filterName: 'html' },
```

File: web/src/repogroups/HomepageConfig.ts [hunk 4]
```diff
@@ -25,7 +25,7 @@ export const homepageLanguageList: { name: string; filterName: string }[] = [
     { name: 'Haskell', filterName: 'haskell' },
```
Comment (felixfbecker) [inline web/src/repogroups/HomepageConfig.ts:25]: ⬆️  Graphql should also be GraphQL

File: web/src/repogroups/HomepageConfig.ts [hunk 5]
```diff
@@ -25,7 +25,7 @@ export const homepageLanguageList: { name: string; filterName: string }[] = [
...
     { name: 'Html', filterName: 'html' },
     { name: 'Java', filterName: 'java' },
-    { name: 'Javascript', filterName: 'javascript' },
+    { name: 'JavaScript', filterName: 'javascript' },
     { name: 'Json', filterName: 'json' },
```
Comment (felixfbecker) [inline web/src/repogroups/HomepageConfig.ts:29]:
```suggestion
    { name: 'JSON', filterName: 'json' },
```

**Comment by sqs** [comment]
Thanks, @felixfbecker, for the other fixes.

**Pull request merged**